### PR TITLE
Ss at view list pretty

### PR DESCRIPTION
**Pull request by snsebro**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description
Updated styling to the view list component, incorporating chakra UI library.

## Related Issue
#15 

## Acceptance Criteria
- [x] The shopping list screen (the screen that lists all of the user’s purchases) looks something like the following (or equivalent, use your creativity!):

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates
Updated layout of view list component to use Chakra UI style library. Updated text and input elements accordingly and changed colors to match brand color direction.
### Before

Screen Shot 2020-11-10 at 9 18 07 PM

### After

Screen Shot 2020-11-24 at 7 03 01 PM

### CODE REVIEW

File: src/components/ViewList.js [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
 import { ListContext } from '../context/ListContext';
-import { Link } from 'react-router-dom';
+import { Link as RouterLink } from 'react-router-dom';
 import calculateEstimate from '../lib/estimates';
-import { Button } from '@chakra-ui/react';
-
+import {
+  Button,
+  Box,
+  Checkbox,
```
Comment (rbompiani) [inline src/components/ViewList.js:8]: Checkbox and List imports are not used and can be removed

File: src/components/ViewList.js [hunk 2]
```diff
@@ -90,17 +113,15 @@ const ViewList = () => {
...
-        .doc(e.target.id)
-        .delete()
-        .then(function () {
-          console.log('Document successfully deleted!');
-        })
-        .catch(function (error) {
-          console.error('Error removing document: ', error);
-        });
-    }
+    currentList.itemsRef
```
Comment (rbompiani) [inline src/components/ViewList.js:117]: I'm relatively new to useRef - can you explain how this works in our standup?

**Review by rbompiani — COMMENTED** [review]
Looks great!  Ya'll are using some fancy useRef footwork in the ViewList component - I can't wait to hear more about how that's working.

**Comment by rachel-fischoff** [comment]
Looks great! A few comments :)
1) It's hard to see the input placeholder with the colored input bar. Maybe we can change the text color for accessibility 
2) the close icon looks a little big. Maybe we can make it smaller? or have space between it and the search bar. It's just a little clunky 
3) the way the gradient is set up now and the way that we re-arranged the nav will look like the picture below 
4) the console is giving warning for the props on the input on lines 202-216
Screen Shot 2020-11-25 at 11 25 25 AM

**Comment by rachel-fischoff** [comment]
I also tried to add a new item from an empty list and it looks like this 
Screen Shot 2020-11-25 at 11 32 50 AM

File: src/components/ViewList.js [hunk 3]
```diff
@@ -114,67 +136,157 @@ const ViewList = () => {
...
+
+        <IconButton
+          colorScheme="blue"
+          aria-label="clear search bar"
+          icon={<CloseIcon />}
+          onClick={handleClearClick}
+        />
+
+        <UnorderedList listStyleType="none">
+          <SimpleGrid columns={3}>
```
Comment (indiragp) [inline src/components/ViewList.js:179]: This violates the HTML rule of the li items being only children of OL or UL. It is currently a child of a div. Maybe swapping SimpleGrid with unorderedList could address this.

Comment (indiragp) [reply src/components/ViewList.js:179]: Try running this chrome extension for accessibility testing: https://accessibilityinsights.io/docs/en/web/overview/ 
There are 2 other errors about having duplicate id's and some elements missing a label (I think the buttons for delete and details)

File: src/style/smartListTheme.js [hunk 4]
```diff
@@ -28,12 +30,18 @@ const smartListTheme = extendTheme({
       lineHeight: '110%',
       letterSpacing: '-2%',
       fontFamily: 'Montserrat',
-      background: 'brand.600',
+      bgImage: 'linear-gradient(to bottom,#76E4F7, #00A3C4)',
```
Comment (indiragp) [inline src/style/smartListTheme.js:38]: I feel like I will get a headache with that gradient, but it is probably just me.

**Pull request merged**